addons/website/views/snippets/snippets.xml

```diff
@@ -411,6 +411,52 @@
          data-selector='.accordion > .card'
          data-drop-in='.accordion:has(> .card)'/>
 
+    <div data-js="masonry_display"
+        data-selector=".s_masonry_block">
+        <we-select string="Template">
+            <we-button title="Default"
+                       data-edit-layout="default"
+                       data-variable="default"
```

Comment:
Any reason you added the variables back? If you want to keep track of which template is selected, you should probably mark it as a data-attribute on the section using `select-data-attribute`.